src/MetaMorpho.sol

```diff
@@ -737,6 +744,8 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
 
                 marketConfig.enabled = true;
 
+                _updateLastTotalAssets(lastTotalAssets + MORPHO.expectedSupplyAssets(marketParams, address(this)));
```

Comment:
Should there be a check that reverts if the market is broken?

`MORPHO.expectedSupplyAssets` won't revert if `elapsed == 0` (and the IRM is broken) and it does not use the `oracle` (called once you need to withdraw)

Reply:
I think it's ok not to revert in this case. Vaults curators should not list broken markets in the first place.